### PR TITLE
Patch: add feature tags

### PR DESCRIPTION
**Pull request by mkholjuraev**

### CODE REVIEW

**Comment by mkholjuraev** [comment]
@ryelo @allieh782 can you please have a look at this and merge if it is ready, thanks

**Comment by allieh782** [comment]
@ryelo given the recent updates to the config, want you to have a look before merging?

File: data/config.yml
Comment (ryelo) [inline data/config.yml:109]: `[data-ouia-app-id="dashboard"]` is not needed in these, this is taken care of by the `scope` above.

```suggestion
        - '{} .ins-c-dashboard__card--PatchManager a:contains("systems affected")'
```

**Review by ryelo — APPROVED** [review]

**Comment by mkholjuraev** [comment]
@ryelo thanks for the PR review. I have found out that Pendo is not collecting information about Patch advisories page. I have checked the config.yaml, everything looks ok. Can you tell me what might be the problem? 
patch

**Comment by mkholjuraev** [comment]
@ryelo thanks for the review. @allieh782 , @ryelo  can we please merge this, if it is ok? Today we have a demo and our team is interested to see the Pendo improvements :)

**Pull request merged**
